### DISCUSSION TITLE
How to change colors in provided template "app-down"

### DISCUSSION THREAD

**Discussion by blaugrau90** [tarampampam/error-pages#146] [Q&A] [answered]
I would like to change the colors in this template, but I don't know how to do it and need support. Can anyone help me here?

**Comment by tarampampam** [comment, marked as answer]
Yes, sure! You can change the colors here on your needs:

https://github.com/tarampampam/error-pages/blob/8e21be0340e3f104cc70a142e8f5a7cf806cbffa/templates/app-down.html#L17-L20

And after that just follow [these instructions](https://github.com/tarampampam/error-pages/wiki/Generator)

**Reply by blaugrau90** [reply]
I have already found the template as well, however I am new to the Docker. I am currently using your Docker image in a Docker-compose file. I can't edit this just like that, can I? And if so how? What about the language? I want to use German texts.
Sorry if this sounds a bit stupid... as I said I'm a bit new to this topic.

**Reply by tarampampam** [reply]
As you probably know - docker images are immutable. You can't edit the original image on your side, but you can build your own image with the required changes, based on our docker image.

In your case, you need to do the following steps:

- Download the required template ([`app-down`](https://github.com/tarampampam/error-pages/blob/master/templates/app-down.html) in your case) on your computer and save with the same name - `app-down.html`
- Edit it on your needs
- Prepare a new `Dockerfile` with the following content (not tested by me, but the basic idea should be clear):

```dockerfile
FROM tarampampam/error-pages

# copy modified template
COPY ./app-down.html /opt/templates/app-down.html

# set your template as the default
ENV TEMPLATE_NAME="app-down"
```

- Build the image (`docker build --tag tour:name .`)
- And now you can use the docker image with your modified template!

Any questions? 😄

**Reply by blaugrau90** [reply]
Very good! That worked! Thank you very much! And big praise to you and the project! Really very cool job!

**Reply by tarampampam** [reply]
You're welcome! And I forgot - separate image building is not required in your case, you can just mount a modified template:

```bash
$ docker run \
  --rm \
  -v "$(pwd)/app-down.html:/opt/templates/app-down.html:ro" \
  -e "TEMPLATE_NAME=app-down" \
    tarampampam/error-pages
```
